docs/logging-usage.rst

```diff
@@ -205,13 +205,74 @@ Delete a metric:
    >>> metric.exists()  # API call
    False
 
-
 Export log entries using sinks
 ------------------------------
 
 Sinks allow exporting entries which match a given filter to Cloud Storage
 buckets, BigQuery datasets, or Cloud Pub/Sub topics.
 
+Export to Cloud storage
```

Comment:
@daspecster You should capitalize `storage` here (if it pleases you to write a new PR)